communities/plymouth-university/community-assessment.md

```diff
@@ -0,0 +1,122 @@
+
+## Goals
+
+### Why do people get involved in the community?
+
+One of the biggest across the university seems to be obtaining a 1 year internship for their "sandwich year", or for getting a job after their degree. Due to the games students being most involved, they also seem to get move involved when the content is likely to contribute towards current assignments.
+
+The one of the problem with GDSoc is that most of the Games Dev course content is based on Unity3D, however this is quite an uncommon technology to use outside of mobile or indie games. Web content seems to cause heavy disengagement from GDSoc members as there seems to be a level of elitisim that attempts to stray away from web based games.
+
+### What activities and goals can we set for the community based on existing goals?
+
+One of the goals we could look towards is trying to find to put on technical talks by experts which relate heavily to lecture content. This has been hard due to the lack of industry in and around Plymouth but it should be a goal to strive towards.
+
+## Location
+
+Due to the nature of the community, the key location for all events and meets is the Plymouth University campus. Any room on campus can be booked out by a students union registered society, this means the GDSoc can book rooms with 2 weeks notice.
+
+The bookable space is very flexible ranging from high spec computer labs which can accomodate up to around 30 or 40 people, over to large lecture theatres which can seat around 200-300. These spaces can be booked for anytime they're not already booked between the hours of 9am and 6pm. After that we need to find a different space.
+
+Lecture theatres are generally very good for speaker based events as the chairs are often quite comfortable and suitable for longer periods of time. They also fit fire restrictions for large groups of people. The computer labs are perfect for small workshops where people can log on and make use of the computers. The chairs in these rooms are also quite comfortable but not always great for talk focused sessions.
+
+There are many pubs with bigger spaces which can be booked for events and these tend to fit up to around 40 people. At the same time smaller meets can happen in the many cafes or pubs around the city and campus.
+
+## Communication
+
+* Facebook is the best medium along with websites such as EventBrite for organising wider events like Game Jams where we invite external bodies to join.
+* We've tried services like Slack and Discord before which have worked somewhat in the past with realtive daily activity but over the past 2 years it's proved very ineffective.
+* Mailchimp has been great for announcing longer term schedules straight to inboxes, we've found open rates are very high but engagement is no different to facebook groups.
```

Comment:
I think this is because we target using university emails and people tend to open those in case there's something important there. The emojis in the title probably help as well.